src/main/java/com/redhat/management/approval/ApprovalApiHelper.java

```diff
@@ -67,12 +67,17 @@ public static Stage findStageByGroup(Group group, List<Stage> stages) {
 
     // Used in bpmn
     public static String getStageContent(String decision) {
-        return decision.equals("denied") ? SKIP : NOTIFY;
+        return (decision.equals("denied") || decision.equals("canceled")) ? SKIP : NOTIFY;
```

Comment:
should call `isStageSkipped` method, and consider renaming it to `isStageSkippable`